src/game.py

```diff
@@ -0,0 +1,33 @@
+from carcassonne_engine import GameEngine, TileSet
+from carcassonne_engine.models import SerializedGameWithID
+from carcassonne_engine.requests import PlayTurnRequest
+
+from .agents import Agent
+from .dispatch import GameDispatch
+from .tokens import Timeout
+from .tree import Tree
+from .utils import assert_no_exception
+
+
+class Game:
+    def __init__(self, engine: GameEngine, tileset: TileSet):
+        self.engine: GameEngine = engine
+        self.tileset: TileSet = tileset
+
+    def run(self, *agents: list[Agent]):
```

Comment:
https://typing.readthedocs.io/en/latest/spec/callables.html#annotating-args-and-kwargs
```suggestion
    def run(self, *agents: Agent):
```